### PR TITLE
[ADDED] Option setter for the URL client connects to

### LINKED ISSUE #94: Problem in connect to remote stan server
Hi, i am trying to connect go stan client to remote stan server and server tell me something like that 
" 2016/07/16 21:48:31 Can't connect: nats: no servers available for connection.
Make sure a NATS Streaming Server is running at: nats://10.0.1.60:4222
"

steps what i am do:
1. make file seed.conf who look like 
"
listen: 0.0.0.0:4222
http: 8222
 "

2 start  server with command:
"./nats-streaming-server -c seed.conf"
who tell me
"
[6763] 2016/07/16 22:52:05.256046 [INF] Starting nats-streaming-server[test-cluster] version 0.2.0
[6763] 2016/07/16 22:52:05.256521 [INF] Starting nats-server version 0.9.0.beta
[6763] 2016/07/16 22:52:05.256544 [INF] Starting http monitor on :8222
[6763] 2016/07/16 22:52:05.256643 [INF] Listening for client connections on 0.0.0.0:4222
"
3. I want to send message to server from other computer whith standart instruction on site
go run stan-pub.go -c "nats://10.0.1.60:4222" foo "msg one" 
and take a message : 
Make sure a NATS Streaming Server is running at: nats://10.0.1.60:4222

what am i doing wrong?

### PR DESCRIPTION
**Pull request by kozlovic**
Also fixed the pub and sub example to use this option setter. They
would otherwise only be able to connect to localhost:4222.

Resolves #94

### CODE REVIEW

**Comment by kozlovic** [comment]
@ColinSullivan1 @aricart Please review at your earliest convenience on Monday.
@mcqueary We may want to issue a new release after this PR is merged. There has been few other option setters added and fixes since `v.0.1.0`.

**Comment by kozlovic** [comment]
@EduardFazliev No because we use `.` for the import, so we don't need to specify `stan`. Not sure why this example has been written this way as opposed to the `pub` one. Will update as part of a different PR.

File: examples/stan-pub.go [hunk 1]
```diff
@@ -31,14 +31,13 @@ func usage() {
 }
 
 func main() {
-	opts := stan.DefaultOptions
-
 	var clusterID string
 	var clientID string
 	var async bool
+	URL := stan.DefaultNatsURL
```
Comment (ColinSullivan1) [inline examples/stan-pub.go]: is this necessary with the flags below?

**Comment by ColinSullivan1** [comment]
LGTM.

File: examples/stan-sub.go [hunk 2]
```diff
@@ -57,10 +55,12 @@ func main() {
 	var qgroup string
 	var unsubscribe bool
 
+	URL := DefaultNatsURL
```
Comment (ColinSullivan1) [inline examples/stan-sub.go]: Same as above...  Nitpicky, I know. ;)

File: examples/stan-pub.go (hunk 1 above)
Comment (kozlovic) [inline examples/stan-pub.go]: Good point, will use var instead.

File: examples/stan-sub.go (hunk 2 above)
Comment (kozlovic) [inline examples/stan-sub.go]: Will change it to var. Thanks!

**Comment by aricart** [comment]
LGTM!

**Pull request merged**